src/devices/corsair_void.c

```diff
@@ -115,7 +115,7 @@ static int void_request_battery(hid_device* device_handle)
         return BATTERY_CHARGING;
     }
 
-    if (data_read[4] == 1) {
+    if (data_read[4] == 1 || data_read[2]) {
```

Comment:
But then this check must be
`if (data_read[2])`

because data_read[4] == 1 is redundant

Reply:
I made a typo I meant to write data_read[4] == 2